docs/move/move-on-aptos/move-security-guidelines.md

```diff
@@ -0,0 +1,574 @@
+---
+title: "Move Security Guidelines"
+---
+
+# Move Security Guidelines
+
+# Introduction
+
+The Move language is designed with security and inherently offers several features including a type system and a linear logic. Despite this, its novelty and the intricacies of some business logic mean that developers might not always be familiar with Move's secure coding patterns, potentially leading to bugs.
+
+This guide addresses this gap by detailing common anti-patterns and their secure alternatives. It provides practical examples to illustrate how security issues can arise and recommends best practices for secure coding. This guide aims to sharpen developers' understanding of Move's security mechanisms and ensure the robust development of smart contracts.
+
+# Access Control
```

Comment:
considering the right column indexes from H2 onward, would it make sense to shift all the headers by 1? so `Access Control` is `##`, `Object Access Control` is `###` and so forth